projects/go-lib/src/lib/components/go-tab/go-tab-group.component.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../../styles/variables';
+
+.go-tab__header {
+  border-radius: $global-radius $global-radius 0 0;
+  display: inline-flex;
+  font-size: 1.125rem;
+  font-weight: normal;
```

Comment:
Do we need to specify this?

Reply:
Ya otherwise it uses the light font-weight. I am changing this to use the variable for this tho